gammapy/utils/modeling.py

```diff
@@ -350,6 +350,25 @@ def set_parameter_covariance(self, covariance, covar_axis):
 
         self.covariance = covariance_new
 
+    # TODO: this is a temporary solution until we have a better way
+    # to handle covariance matrices via a class
+    def error(self, parname):
+        """
+        Return error on a given parameter
+
+        Parameters
+        ----------
+        parname : str
+            Parameter
+        """
+        if self.covariance is None:
+            raise ValueError('Covariance matrix not set.')
+
+        for i, parameter in enumerate(self.parameters):
+            if parameter.name == parname:
```

Comment:
This lookup of parameter by name could / should be a separate method that takes the name and returns the int index, no? If you don't want to implement that here, OK to merge anyways of course.